src/components/PostInfo/PostInfo.tsx

```diff
@@ -1,5 +1,29 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { UserInfo } from '../UserInfo';
+import { CommentList } from '../CommentList';
+import './PostInfo.scss';
 
-export const PostInfo: React.FC = () => (
-  <>Put the post here</>
+type Props = {
+  post: Post
+};
+
+export const PostInfo: React.FC<Props> = ({ post }) => (
+  <div className="PostInfo" key={post.id}>
```

Comment:
Remove the key prop from the PostInfo component, as it is not needed here. The key prop should be applied to the component when it is rendered in a list, not within the component itself.
```suggestion
  <div className="PostInfo" >
```